File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/save-unit-commander_item.ts


import { ISaveUnitCommander_Item } from './isave-unit-commander_item';

export class SaveUnitCommander_Item implements ISaveUnitCommander_Item {
  
	public PersonID?: number;
	public UnitID: number = 0;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public Gender: string = "";
	public ModifiedByAppUserID: number = 0;
  
}

File: GTTS/INL.UnitLibraryService.Models/SaveUnitCommander_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class SaveUnitCommander_Item : ISaveUnitCommander_Item
    {
        public long? PersonID { get; set; }
        public long UnitID { get; set; }
        public string FirstMiddleNames { get; set; }
        
        
        
        public string LastNames { get; set; }
        public char Gender { get; set; }
        public int ModifiedByAppUserID { get; set; }
    }
}
